ui/src/frontend/track_panel.ts

```diff
@@ -125,8 +127,13 @@ class TrackShell implements m.ClassComponent<TrackShellAttrs> {
   private attrs?: TrackShellAttrs;
   private defaultHeight?: number;
 
+  private transparentImage?: HTMLImageElement;
+
   oninit(vnode: m.Vnode<TrackShellAttrs>) {
     this.attrs = vnode.attrs;
+
+    this.transparentImage =new Image();
```

Comment:
nit. And not caught by lint?
```suggestion
    this.transparentImage = new Image();
```